components/ome-jxr/src/ome/jxr/ifd/IFDContainer.java

```diff
@@ -38,7 +38,9 @@
  * <dl>
```

Comment:
I wonder why we keep starting these description lists. Not this PR's fault though.